mongodb_proxy.py

```diff
@@ -83,7 +83,7 @@ def __getitem__(self, key):
         Otherwise just return the attribute.
         """
         item = self.proxied_object[key]
-        if hasattr(item, '__call__'):
+        if hasattr(item, '__call__') or attr.__class__.__name__ in ['Database']:
```

Comment:
https://github.com/mongodb/mongo-python-driver/pull/542/files

This is temporary fix to run the tests.